### PR TITLE
Feat/scl (fist review and merge the other one)

### PR DESCRIPTION
**Pull request by tomADC443**
The PR adds functionality to display SCL data. The component needs a bit more styling though.

### CODE REVIEW

File: src/components/atoms/BackButton/BackButton.tsx
Comment (joostwmd) [inline src/components/atoms/BackButton/BackButton.tsx:1]: i think we should not zoom out the map entirely when the user clicks back. It is a bit tedious to zoom in again when you wanted to select a nearby AOI

File: src/components/atoms/SceneClassification/SceneClassButton.tsx
Comment (joostwmd) [inline src/components/atoms/SceneClassification/SceneClassButton.tsx:1]: Why is the component called MapProjectionButton? From looking at the code this is the component that handles the display of Additional Informations like clouds right?

**Review by joostwmd — COMMENTED** [review]
Nice work! Especially the single state for the predictions!

File: src/services/predictionLayerService.tsx [hunk 1]
```diff
@@ -117,27 +152,22 @@ export function getBoundingBox(polygon: Polygon): [number, number, number, numbe
...
-            console.log('removing layer', layer.id)
-            map.removeLayer(layer.id)
-        }
-    })
+    const mapStyle = map.getStyle()
+    if (mapStyle.layers) {
+        const layers = mapStyle.layers
+        layers.forEach((layer) => {
+            if (layer.id.startsWith('prediction')) {
+                console.log('removing layer', layer.id)
```
Comment (emely3h) [inline src/services/predictionLayerService.tsx]: can be removed

Comment (tomADC443) [reply src/services/predictionLayerService.tsx]: true! good catch

Comment (tomADC443) [inline src/services/predictionLayerService.tsx]:
```suggestion
           
```

File: src/components/atoms/BackButton/BackButton.tsx
Comment (tomADC443) [reply src/components/atoms/BackButton/BackButton.tsx:1]: Thats a good note i will write a ticket for it.

File: src/components/atoms/SceneClassification/SceneClassButton.tsx
Comment (tomADC443) [reply src/components/atoms/SceneClassification/SceneClassButton.tsx:1]: Thats a good catch! Its a copy paste leftover. I rename the button.

**Pull request merged**